static/app/views/settings/organizationRateLimits/organizationRateLimits.tsx

```diff
@@ -47,84 +51,107 @@ function OrganizationRateLimit({organization}: OrganizationRateLimitProps) {
   };
 
   return (
-    <div>
-      <SettingsPageHeader title={t('Rate Limits')} />
+    <Feature
+      organization={organization}
+      features={['legacy-rate-limits']}
+      renderDisabled={_ => (
+        <EmptyMessage
+          icon={<IconInfo size="xl" />}
+          title={t(`Legacy rate limits are not available in this organization.`)}
+        >
+          {tct(
+            `See our [link:Billing Quota Management] docs for more information about managing your organization's quota.`,
```

Comment:
you might have to put this sort of thing in getsentry still and use a hook to bring it in? `feature-disabled:dashboards-page` would be an example.

Reply:
noted! we're still finalizing copy so i'll wait for that before determining whether to merge as is or do the hook thing instead